cmd/gateway-main.go

```diff
@@ -106,6 +106,7 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 	logger.Disable = true
 
 	// Validate if we have access, secret set through environment.
+	globalGatewayName = gw.Name()
```

Comment:
gatewayName can be deleted and references  in this file replacedwith globalGatewayName